src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -185,6 +185,9 @@
    :component-will-mount
    (fn [{:keys [this]}]
      (this :-refresh-workspace))
+   :component-will-receive-props
+   (fn [{:keys [this after-update]}]
+     (after-update this :-refresh-workspace))
```

Comment:
Does this after-update even do anything? Can we just `(this :-refresh-workspace)`?